nixos/modules/services/web-servers/zope2.nix

```
@@ -52,7 +52,7 @@ let
             </blobstorage>
           </zodb_db>
           '';
-        type = types.string;
+        type = types.str;
```

it is kinda XML, but it is still extensible, accroding to https://github.com/kiorky/plone2/blob/master/etc/zope.conf.in
```suggestion
        type = types.lines;
```